src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -33,24 +33,50 @@ import org.dia.core.{SciSparkContext, SciTensor}
  */
 class GTGRunner(val masterURL: String,
                 val paths: String,
-                val partitions: Int) {
```

Comment:
Can the variable name(s) be also included as an input parameter?
Reasoning: consider the use case where a difference dataset is used, or the end user has modified the input data some how leveraging NCO [http://nco.sourceforge.net/] operators for e.g. and has renamed the variable of interest in original the file. If this is too big a change for this PR, consider updating the doc string in GTG runner to indicate that the variable must be called 'ch4' in this PR, then create the new issue.